frontend/src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { AppHeaderWrapper } from './AppHeader.styled';
+import { PageHeader } from 'antd';
+
+export interface AppHeaderProps {
+  pageTitle?: string;
+  logo?: React.ReactNode;
+}
+
+export const AppHeader = ({ pageTitle, logo }: AppHeaderProps) => {
+  const providedLogo = logo ? logo : null;
+
+  return (
+    <AppHeaderWrapper>
+      <div>{providedLogo}</div>
```

Comment:
Zastanawiam się czy to logo może być zmieniane czy nie będzie tu na stałe :)

Reply:
Raczej nie będziemy zmieniać